runtimes/eden/src/lib.rs

```diff
@@ -267,7 +267,7 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 
-	impl pallet_contracts_rpc_runtime_api::ContractsApi<Block, AccountId, Balance, BlockNumber, Hash>
+	impl pallet_contracts::ContractsApi<Block, AccountId, Balance, BlockNumber, Hash>
```

Comment:
Without having an rpc_runtime_api, I wonder if `ContractsApi` exists in pallet_contracts or not?

Reply:
it has been moved to the pallet_contract itself. [See](https://docs.rs/pallet-contracts/latest/pallet_contracts/trait.ContractsApi.html)